packages/sdk/src/oracle/services/tests/statemachine.test.ts

```diff
@@ -1,131 +0,0 @@
-import assert from "assert";
```

Comment:
the statemachine refactors means it can no longer be unit tested unfortunately. there are new e2e tests in the client that test the statemachine implicitly as well as the individual contexts.